### PR TITLE
new upstream v3.15.4

### PR DESCRIPTION
**Pull request by joelkraehemann**
* icons with reverse domain notation

### CODE REVIEW

File: org.nongnu.gsequencer.gsequencer.json [hunk 1]
```diff
@@ -210,15 +209,17 @@
                 "make html",
                 "make install-html-mkdir",
                 "make install-html",
-                "sed -i -e 's~Icon=/app/share/gsequencer/icons/ags.png~Icon=org.nongnu.gsequencer.gsequencer.png~g' /app/share/applications/gsequencer.desktop",
+                "sed -i -e 's~Icon=gsequencer~Icon=org.nongnu.gsequencer.gsequencer~g' /app/share/applications/gsequencer.desktop",
+                "find /app/share/icons/hicolor -name 'gsequencer.png' -exec /bin/sh -c 'mv {} $(dirname {})/org.nongnu.gsequencer.gsequencer.png' ';'",
+                "mv /app/share/icons/hicolor/scalable/apps/gsequencer.svg /app/share/icons/hicolor/scalable/apps/org.nongnu.gsequencer.gsequencer.svg",
```
Comment (hfiguiere) [inline org.nongnu.gsequencer.gsequencer.json:214]: not sure why you don't use the `rename-icon` here. It does take care for all of this.

Comment (joelkraehemann) [reply org.nongnu.gsequencer.gsequencer.json:214]: Yeah, just reverted it.

Comment (hfiguiere) [reply org.nongnu.gsequencer.gsequencer.json:214]: note that since you are upstream you might as well give them the proper name in the original source.

Comment (joelkraehemann) [reply org.nongnu.gsequencer.gsequencer.json:214]: Yes, I could but for now I am unsure if I should use org.nongnu.gsequencer.gsequencer or org.gsequencer.gsequencer

Comment (hfiguiere) [reply org.nongnu.gsequencer.gsequencer.json:214]: Right now it is `org.nongnu.gsequencer.gsequencer`, and changing it is a bit more involved.

**Pull request merged**